src/com/hms/flexyosisoftconnector/OSIsoftConfig.java

```diff
@@ -160,6 +169,13 @@ public static void initConfig(String configFile) throws JSONException, IOExcepti
       }
     }
 
+    if (appConfig.has(HTTP_TIMEOUT_SECONDS_KEY)) {
+      Logger.LOG_INFO("http timeout retrieved from configuration file.");
+      httpTimeoutSeconds = appConfig.getInt("httpTimeoutSeconds");

```

Comment:
```suggestion
      httpTimeoutSeconds = appConfig.getInt("httpTimeoutSeconds");
      Logger.LOG_INFO("HTTP timeout of " + httpTimeoutSeconds + 
          " seconds retrieved from configuration file.");
```

Reply:
most users will not use the http timeout so it will usually be the default, not technically from the config file.
I added a log in osisoft config that only logs if it is from the config file. the https on that also needs to be capitalized

Reply:
@oliver-walker-hms  this suggestion is at the spot of that log entry you added, this is just adjusted to capitalize the HTTP and add the value read from the config file.